refractr/schema.yml

```diff
@@ -72,7 +72,7 @@ defs:
     title: dst-url
     type: string
     # single '$' added to file matcher to allow nginx exprs: $lang $request_uri
-    pattern: '^(?!https?:\/\/)([A-z0-9]+([A-z0-9\.\-$]+)\/)([A-z0-9\-_]+\/)*([A-z0-9$\/\.\-_:]+)?(\?[A-z0-9$\.\-:=]+(&[A-z0-9$\.\-:=]+)*)?(#[A-z0-9\-_]+)?\??$' # added optional '?' for nginx rewrite
+    pattern: '^(?!https?:\/\/)([A-z0-9]+([A-z0-9\.\-$]+)\/?)([A-z0-9\-_]+\/)*([A-z0-9$\/\.\-_:]+)?(\?[A-z0-9$\.\-:=]+(&[A-z0-9$\.\-:=]+)*)?(#[A-z0-9\-_]+)?\??$' # added optional '?' for nginx rewrite
```

Comment:
I made the slash between the domain and the path optional here so that I could use `$uri` in the destination, otherwise if would add a double slash if I included the slash before the $ in the destination.